src/fetchAndDecrypt.ts

```diff
@@ -3,23 +3,33 @@ import { createDecipheriv } from 'crypto-browserify';
 
 // local
 import decryptStream from './decrypt';
+import { PenumbraError } from './error';
 import { RemoteResourceWithoutFile } from './types';
 import { toBuff } from './utils';
+import emitError from './utils/emitError';
 
 /**
  * Fetches a remote file from a URL, deciphers it (if encrypted), and returns a ReadableStream
  *
  * @param resource - The remote resource to download
+ * @param fetchOptions - Options to include in the download URL fetch
  * @returns A readable stream of the deciphered file
  */
-export default function fetchAndDecrypt({
-  url,
-  decryptionOptions,
-}: RemoteResourceWithoutFile): Promise<ReadableStream> {
+export default function fetchAndDecrypt(
+  { url, decryptionOptions }: RemoteResourceWithoutFile,
+  fetchOptions: RequestInit = { credentials: 'include' },
```

Comment:
same here, adding this as default ends up resulting in non-backwards comptaibilitiy

Reply:
Ok, inverting the default.